spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -678,6 +655,202 @@ boolean result = pollableSource.poll(received -> {
+If such channel has a subscriber all errors will be sent to that subscriber.
+
+Consider the following:
+
+[source,text]
+----
+spring.cloud.stream.bindings.input.group=myGroup
+----
+
+[source,java]
+----
+@StreamListener(Sink.INPUT) // destination name 'input.myGroup'
+public void handle(Person value)
+	throw new RuntimeException("BOOM!");
+}
+
+@StreamListener(Processor.INPUT + ".myGroup.errors") //channel name 'input.myGroup.errors'
+public void error(Message<?> message) {
+	System.out.println("Handling ERROR: " + message);
+}
+----
+
+The `handle(..)` method which subscribes to the channel named `input` throws an exception.  Given there is also a subscriber to the error channel `input.myGroup.errors`
+all error messages will be handled by this subscriber.
+
+I the event you have multiple bindings you may want to chose to have a single error handler. Spring Cloud Stream automatically provides support for
+_global error channel_ by bridging each individual error channel to the channel named `errorChannel`, allowing a single subscriber to handle all errors.
+
+[source,java]
+----
+@StreamListener(“errorChannel")
+public void error(Message<?> message) {
+	System.out.println("Handling ERROR: " + message);
+}
+----
+
+This may be a convenient option if error handling logic is the same regardless of which handler produced the error.
+
+Also, error messages sent to the `errorChannel` can be published to a specific destination at the broker by configuring a binding named `error` for the outbound target.
+This option provides a mechanism to automatically send error messages to another application bound to that destination or for later retrieval (e.g., audit).
+For example, to publish error messages to a broker destination named `myErrors`, set the following property:
+
+[source,text]
+----
+spring.cloud.stream.bindings.error.destination=myErrors.
+----
+
+===== System Error Handling
+
+System level error handling implies that the errors are communicated back to the messaging system and given that not every messaging system
+is the same the capabilities may differ from binder to binder.
+
+That said, in this section we’ll explain the general idea behind system level error handling and will use Rabbit binder as an example. NOTE: Kafka binder provides similar
+support although some configuration properties do differ. Also for more details and confuration options please refer to individual binder's documentation.
+
+If no internal error handlers are configured the errors are propagated to the binders and binders subsequently propagate those errors back to the messaging system.
+Depending on the capabilities of the messaging systems such system may _drop_ the message, _re-queue_ the message for re-processing or _send the failed message to DLQ_.
+Both Rabbit and Kafka support these concepts. However other binders may not, so please refer to individual binder’s documentation for details on supported system
```

Comment:
Replace with "Both Rabbit and Kafka support these concepts. However, other binders may not, so refer to your individual binder’s documentation for details on supported system-" (Note the hyphen at the end.)